public/docs/_examples/attribute-directives/dart/lib/app_component.dart

```diff
@@ -0,0 +1,13 @@
+// #docregion
+library attribute_directives.app_component;
```

Comment:
TypeScript's directives feature is called "annotations" in Dart. So...

attribute_directives -> attribute_annotations?